docs/source/tutorials/traits_ui_scientific_app.rst

```diff
@@ -121,22 +121,8 @@ A point object could be implemented in python with:
 
     `code snippet #0 <../_static/code_block0.py>`_
```

Comment:
This feels off. We were probably doing this to link back to the code snippet but given that we can now do literalinclude, we can get rid of this.

Reply:
once the links/lines are removed (this and other code snippets), we will also want to check that the files aren't still included in `_static`.